app/models/oidc_provider/access_token.rb

```diff
@@ -7,11 +7,37 @@ class AccessToken < ApplicationRecord
     attribute :token, :string, default: -> { SecureRandom.hex 32 }
     attribute :expires_at, :datetime, default: -> { 1.hours.from_now } 
 
-    def to_bearer_token
-      Rack::OAuth2::AccessToken::Bearer.new(
-        access_token: token,
-        expires_in: (expires_at - Time.now).to_i
+    def to_bearer_token(with_refresh_token)
```

Comment:
Keyword argument for clarity please
```suggestion
    def to_bearer_token(with_refresh_token:)
```